README.md

```diff
@@ -13,18 +13,17 @@
 ## Installation
 On a terminal:
 ```bash
-conda create -n qsui python=3.9
-conda activate qsui
-
-git clone https://github.com/aps-bcda/qsui
+git clone git@github.com:BCDA-APS/qsui.git
 cd qsui
-pip install .
+conda env create --force -n qsui -f ./env.yml
+conda activate qsui
+pip install -e .
```

Comment:
```suggestion
pip install -e . --no-deps
```

Reply:
With this change, pip will not try to install packages already installed by conda.